app/src/main/java/com/kks/nimblesurveyjetpackcompose/viewmodel/home/HomeViewModel.kt

```diff
@@ -0,0 +1,30 @@
+package com.kks.nimblesurveyjetpackcompose.viewmodel.home
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.kks.nimblesurveyjetpackcompose.di.IoDispatcher
+import com.kks.nimblesurveyjetpackcompose.repo.home.HomeRepo
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@HiltViewModel
+class HomeViewModel @Inject constructor(
```

Comment:
Surely this should be done on an integration ticket? Currently the fetched data is not being used anywhere 🤔